src/en/news/blog/2021/2021-ceph-user-survey-results/index.md

```diff
@@ -10,8 +10,8 @@ The purpose of this survey is to better understand how Ceph technologies have be
 
 You can find both the report and raw data below.
 
-- [2021 Ceph User Survey Results](https://ceph.io/wp-content/uploads/2021/05/Ceph-User-Survey-2021.pdf) 
-- [2021 Ceph User Survey Raw Data](https://ceph.io/wp-content/uploads/2021/05/Ceph-User-Survey-2021-raw-results.csv) 
+- [2021 Ceph User Survey Results](attachments/Ceph-User-Survey-2021.pdf) 
+- [2021 Ceph User Survey Raw Data](attachments/Ceph-User-Survey-2021-raw-results.csv) 
```

Comment:
@adamduncan Here's a weird one.  Sage's change doesn't render at https://fix-survey-links.ceph.io/en/news/blog/2021/2021-ceph-user-survey-results/.  Those links are still the old wordpress URL.  Is there some wizardry going on in the compilation that is ignoring the change because there's no FQDN or something?

Reply:
@djgalloway We'll need to specify those asset types in our config. By default we process most image types ([source](https://github.com/ceph/ceph.io/blob/main/scripts/images.js#L16)), which ensures they end up in the `dist` build for uploading to the web server.

Other non-image assets (e.g. `.pdf`, `.csv`) will need to be explicitly "passed through" to the build output ([`addPassthroughCopy`](https://www.11ty.dev/docs/copy/) is Eleventy's mechanism for this). 

We currently do this for `.json` assets ([config source](https://github.com/ceph/ceph.io/blob/main/.eleventy.js#L86)). Looks like we'll want to do the same for `pdf` and `csv` from within the site too?

1. Do we expect we'll need to pass these through just for Blog posts, or do we foresee other sections of the site linking to local assets that'll also need to end up in build?
1. Are there any other file types you expect we might need in future?

Reply:
@Thingee Can we get your input, please?  Neither of those questions are my wheelhouse really :smile:

Reply:
All good. I think let's go with 1. assuming they could exist throughout the site (i.e. not just blogs), and 2. we can always opt-in to add other file types as we encounter them.

I'd suggest we add:

```js
eleventyConfig.addPassthroughCopy('./src/**/*.{csv,pdf}'); 
```

Happy to include that here, as it's the first instance of supporting these file types from content in the site. Would you like me to add that to this PR, @djgalloway?

Reply:
That's fine with me.  Thank you!

Reply:
No problem. Added with 4b6bd42